lms/djangoapps/verify_student/views.py

```diff
@@ -1251,3 +1274,29 @@ def _track_reverification_events(self, event_name, user_id, course_id, checkpoin
                     }
                 }
             )
+
+    def _redirect_no_initial_verification(self, user, course_key):
+        """Redirect because the user does not have an initial verification.
+
+        NOTE: currently, we assume that courses are configured such that
+        the first re-verification always occurs AFTER the initial verification
+        deadline.  Later, we may want to allow users to upgrade to a verified
+        track, then submit an initial verification that also counts
+        as a verification for the checkpoint in the course.
+
+        Arguments:
+            user (User): The user who made the request.
+            course_key (CourseKey): The identifier for the course for which
+                the user is attempting to re-verify.
+
+        Returns:
+            HttpResponse
+
+        """
+        log.error(
```

Comment:
@wedaly did you mean for this to become an error?   in the previous commit you had downgraded to `info` IIRC.